### DISCUSSION TITLE
请继续关注我们的韩国客户。

### DISCUSSION THREAD

**Discussion by xczsonn** [MaaAssistantArknights/MaaAssistantArknights#4001] [General]
在韩国,智能基建换班、效率低下,"选择JSON文业、选择亲笔"无法正常启动,直接停止了工作。 这是只要稍微测试一下就能知道的东西。 请特别注意。

ps. 还有，关于自定义基建排班json共享网站吗?

**Comment by liuyifan-eric** [comment]
The machine translation is hard to understand. Could you provide the original Korean text and the translated English text? (Or it would be even better if you speak English)

The project is open source, non-profit, and maintained by volunteers. If you would like to help us, you can contribute to the project (we are working on updating the tutorials)

**Reply by liuyifan-eric** [reply]
기계 번역은 이해하기 어렵습니다. 한글 원본과 번역된 영문 원본을 제공해주실 수 있나요? (아니면 영어를 할 수 있다면 더 좋을 것입니다)

이 프로젝트는 오픈 소스이며 비영리이며 자원 봉사자들에 의해 유지 관리됩니다. 우리를 돕고 싶다면 프로젝트에 기여할 수 있습니다(튜토리얼 업데이트 작업 중).

**Reply by xczsonn** [reply]
한국에서는 스마트 인프라 교체 기능이 매우 비효율적으로 작동하고 '选择作业 JSON 文件，自动抄作业'이 인게임에서 오퍼레이터를 제대로 배치하지 않습니다.
ps. 그리고 커스텀 기반시설 json 공유 사이트가 있습니까? json생성기는 중국어만 지원해서 제대로 사용할 수가 없습니다 한국어를 지원해주십시오 ㅠㅠ






In Korea, the smart infrastructure replacement function is very inefficient and ''选择作业 JSON 文件，自动抄作业' does not properly deploy the operator in the game. 
ps. And is there a json shared site of custom infrastructure? I don't know because json generator only supports Chinese. Please support Korean.

**Comment by MistEO** [comment]
Infrast builder: https://yituliu.site/riicCal

But it does not support other languages, I raised an issue and waiting for the developers to reply. https://github.com/Zirun-wang/yituliuFrontEnd/issues/20

**Reply by xczsonn** [reply]
신경 써주셔서 감사합니다. 영어 지원이 된다면 한국어를 비롯한 다른 언어의 번역도 빠르게 가능할 것입니다.

Thank you so much for your care. If English is supported, it will be possible to translate Korean and other languages quickly.

**Comment by MistEO** [comment]
I'm sorry, it's not that we don't care about the KR client, but as you can see, MAA has been overloaded with issues and we don't have the manpower to deal with them.

MAA is an open source, non-profit project and if anyone would like to join us, we would be very grateful. 
Adaptation to overseas clients requires only screenshots and json changes in most cases. If you want to try it, you may refer [here](https://maa.plus/docs/en-us/2.5-OVERSEAS_CLIENTS_ADAPTATION.html).

**Reply by xczsonn** [reply]
답변해주셔서 감사합니다 현재 다른 서버에서 altnate 캐릭터가 원본 캐릭터와 동시에 배정되지 않아 충돌하는 문제가 생기는 것은 유감이며 빠른 시일 내에 고쳐졌음 합니다. 이 방법을 한국어 커뮤니티에 공유합니다.

Thank you for your answer It's a shame that altnate characters are currently not assigned at the same time as the original characters on other servers, causing conflicts, and hopefully they'll be fixed soon. Share this method with the Korean community.

**Reply by xczsonn** [reply]
이 문제는 이미지의 modify와 관련이 없는 것 같습니다. 그리고, 기반시설, custom 기반시설에서 '泡泡'를 인식 못하는 것도 resource 폴더에서 찾아볼 수 없었습니다.

I don't think this issue is related to the modify of the image. In addition, infrastructure, custom infrastructure, the inability to recognize '泡泡' was not found in the resource folder.
